### ISSUE TITLE
Should you be able to edit ports & env vars for a service?

### ISSUE THREAD

**Issue by jem-computer** [yldio/paz-web#13]
Looks like there's just form fields for public facing & # of instances. I'm sure I remember being able to edit env vars.

``` handlebars
          {{#if configNext.isEditing}}
            {{#form-wrapper classNames='config__form'}}
              {{input configNext.publicFacing label="Public Facing?" as="checkbox"}}
              {{input configNext.numInstances label="Instances"}}
              <button class='btn btn--primary btn--sm' {{action 'save' this 'configNext'}}>
                <span class="glyphicon glyphicon-pencil"></span>
                Save
              </button>
              <button class='btn btn--sad btn--sm' {{action 'cancel' this 'configNext'}}>
                Cancel
              </button>
            {{/form-wrapper}}
          {{else}}
            {{service-config config=configNext}}
          {{/if}}
```

![pazember 2015-03-19 12-33-34](https://cloud.githubusercontent.com/assets/591643/6730411/435856f2-ce34-11e4-8160-6a960df71604.jpg)

**Comment by lukebond** [comment]
I don't recall it ever being editable. It was probably one of those "don't strictly need it for CC in 09/2014 so it's of secondary importance" things. But yeah we do need it to be added at some point.

**Comment by lukebond** [comment]
Fixed by PR #23
